### PR TITLE
feat(web): setting page

### LINKED ISSUE #6: [2JH] Settings UI
## Describe the issue 
Managers should be able to update the settings of the application, per figma specifications

## Resource request
N/A

## Definition of done 

- [ ] TBD 

## Testing plan
should be deployed on prod without any issues. no automated testing required

## JEH
2

### PR DESCRIPTION
**Pull request by lambdina**
# Description

Please provide a detailed description of what was done in this PR.
Precise the issue that you are resolving.

# Fixes

Specify which issue this PR targets, like so:

- fixes #6

### CODE REVIEW

File: web/src/pages/Setting/set-team.tsx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        onRequestClose();
+    };
+
+    return (
+        <Modal
+            className="popup"
+            isOpen={isOpen}
+            onRequestClose={onRequestClose}
+        >
+            <h2>Inviter un nouveau membre par email</h2>
```
Comment (djnnvx) [inline web/src/pages/Setting/set-team.tsx]: this should be in english

File: web/src/pages/Setting/set-security.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            </label>
+            <input
+                type="password"
+                id="confirmPassword"
+                value={confirmPassword}
+                onChange={(e) => setConfirmPassword(e.target.value)}
+            />
+            <br />
+            <button type="submit" onClick={handleChangePassword}>
+                Modifier le mot de passe
```
Comment (djnnvx) [inline web/src/pages/Setting/set-security.tsx]: this should be in english

File: web/src/pages/Setting/set-security.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        // Remarque : ceci est un exemple simplifié à des fins de démonstration uniquement
+        if (currentPassword === 'currentpassword') {
+            alert('Le mot de passe a été modifié avec succès!');
+        } else {
+            alert('Le mot de passe actuel est incorrect.');
+        }
+    };
+
+    const handleToggleTwoFactor = () => {
+        setIsTwoFactorEnabled(!isTwoFactorEnabled);
```
Comment (djnnvx) [inline web/src/pages/Setting/set-security.tsx]: 2FA is not supported yet in the backend, this is safe to comment

File: web/src/pages/Setting/set-security.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    const handleChangePassword = () => {
+        // Vérifier que les champs de mot de passe ne sont pas vides et que le nouveau mot de passe correspond à la confirmation
+        if (
+            !currentPassword ||
+            !newPassword ||
+            !confirmPassword ||
+            newPassword !== confirmPassword
+        ) {
+            alert(
+                'Veuillez remplir tous les champs de mot de passe et vous assurer que les mots de passe correspondent.'
```
Comment (djnnvx) [inline web/src/pages/Setting/set-security.tsx]: this should be in english

File: web/src/pages/Setting/set-team.tsx [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+                />
+                <button type="submit">Inviter</button>
+            </form>
+        </Modal>
+    );
+}
+
+function SettingTeam({ userId, userRole }: UserGroupsProps) {
+    const [groups, setGroups] = useState<Group[]>([]);
+    const [maClasse, setMaClasse] = useState('action-ctn');
```
Comment (djnnvx) [inline web/src/pages/Setting/set-team.tsx]: what is this supposed to be ?

**Review by djnnvx — CHANGES_REQUESTED** [review]
please fix these comments @samacamelia

File: web/src/pages/Register/signup.scss
Comment (claraxuxu) [inline web/src/pages/Register/signup.scss:1]: Ce fichier n'existe plus depuis longtemps sur next, faut delete

File: web/src/pages/Register/SignUp.tsx
Comment (claraxuxu) [inline web/src/pages/Register/SignUp.tsx:1]: fichier not updated...API est déjà intégré

**Review by djnnvx — APPROVED** [review]

**Pull request merged**